src/BunqLib.php

```diff
@@ -245,6 +245,30 @@ public function makePayment(
         )->getValue();
     }
 
+    /**
+     * @param string $amount
+     * @param string $recipient
+     * @param string $description
+     * @param string $name
+     * @param MonetaryAccountBank $monetaryAccount
+     *
+     * @return int
+     */
+    public function makePaymentIBAN(
+        string $amount,
+        string $recipient,
+        string $description,
+	string $name,
+        MonetaryAccountBank $monetaryAccount
```

Comment:
Why is monetary account required ? As the it is optional for the `Payment::create()` method.

Reply:
It is also required in the makePayment method ;) so yes it seems to be required at least in the tinker libs :)